### ISSUE TITLE
waifu-caffe dead after updated VGA driver

### ISSUE THREAD

**Issue by abdjy001ya** [lltcggie/waifu2x-caffe#11]
hello,
The software is really good,  The waifu2x-caffe(ver1.1.1.1)  was working fine here before, and today after I updated the VGA driver, waifu2x-caffe.exe didn't work anymore.
The situation was: I double clicked the waifu2x-caffe.exe, no windows appeared but still created a process called "waifu2x-caffe.exe" in Task Manager. 
Furthermore, that process was unable to kill in Task Manager.  It was unable to be killed even I use software like lockhunter, killbox etc. And since so, I can't shutdown or reboot my PC because the system was unable to kill it and stuck in the shutting down screen. Cut the power was the onlyway.
I tried to delete the folder, change the path, recover my VGA driver but they didn't work.
Here's my device info
OS:Win7 X64 
VGA:GTX-660 (driver verson 361.75 now)
I was hopeless and wish i may got some help here,thank you!

**Comment by lltcggie** [comment]
The waifu2x-caffe(ver1.1.1.1) has been running without any problems in my PC.
Here's my device info
OS: Win8.1 x64
VGA: GTX 660 (driver verson 361.75 now)
Please try uninstall VGA driver and install old version driver.

Translate by Google Translate

**Comment by abdjy001ya** [comment]
hello, the problem was solved. 
I uninstalled VGA driver and install old version driver but it didn't work.
I reboot & enter safe mode and finally find out the problem is my antivirus(kaspersky KIS 2016).
I totally don't get it. Waife2x-caffe was working fine under KIS 2016, before the VGA driver update.
How is it concern to antivirus after the VGA driver update?
Anyway after I added waifu2x-caffe to it's exception list, the problem was solved. 
Thank you for your help.
